src/otx/algo/detection/ssd.py

```diff
@@ -572,21 +601,42 @@ def _exporter(self) -> OTXModelExporter:
 
         mean, std = get_mean_std_from_data_processing(self.config)
 
-        with self.export_model_forward_context():
-            return MMdeployExporter(
-                model_builder=self._create_model,
-                model_cfg=deepcopy(self.config),
-                deploy_cfg="otx.algo.detection.mmdeploy.ssd_mobilenetv2",
-                test_pipeline=self._make_fake_test_pipeline(),
-                task_level_export_parameters=self._export_parameters,
-                input_size=self.image_size,
-                mean=mean,
-                std=std,
-                resize_mode="standard",
-                pad_value=0,
-                swap_rgb=False,
-                output_names=["feature_vector", "saliency_map"] if self.explain_mode else None,
-            )
+        return OTXNativeModelExporter(
+            task_level_export_parameters=self._export_parameters,
+            input_size=self.image_size,
+            mean=mean,
+            std=std,
+            resize_mode="standard",
+            pad_value=0,
+            swap_rgb=False,
+            via_onnx=True,  # Currently SSD should be exported through ONNX
+            onnx_export_configuration={
+                "input_names": ["image"],
+                "output_names": ["boxes", "labels"],
+                "dynamic_axes": {
+                    "image": {0: "batch", 2: "height", 3: "width"},
```

Comment:
Why do we need dynamic image size here? In MAPI we automatically insert resize graph node, rather than making model recappable.

Reply:
Anchor generator generates default anchor from feature map size, and the feature map size come from image size.
If we don't specify image size as a dynamic, then ONNX convert anchor generator to generate anchors from constant values. As a result, the graph between anchor generator and object detector are broken. So we need to specify this for ONNX conversion.